### PR TITLE
Updates to README.md fore release 8.23.10

### PR DESCRIPTION
**Pull request by niallriddell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,36 @@
...
+***IMPORTANT:***  Please follow the guidelines documented here if you are upgrading from a previous version of React SDK: [Upgrading the SDK](https://docs.pega.com/bundle/constellation-sdk/page/constellation-sdks/sdks/upgrading-sdk.html)
 
-2.	The Pega-provided React SDK components have been moved into an npm package (@pega/react-sdk-components) that is a new dependency for the React SDK. This means that a new React SDK project provides an **empty component structure** that will only contain the SDK user’s code without the clutter of the Pega provided components. SDK users will add their components to the project by creating custom components or overriding existing Pega components. It also allows for easier updates to the Pega-provided components since updates to the npm package won’t directly affect the SDK user’s code.
+If you want to continue using the previous release you can checkout the [release/8.8.10](https://github.com/pegasystems/react-sdk/tree/release/8.8.10)
```
Comment (JEAxero) [inline README.md:21]: add "branch" at the end of the sentence OR remove "the".

File: README.md [hunk 2]
```diff
@@ -1,39 +1,36 @@
...
 
-<h1 style="color: red">
-MAIN BRANCH <u><em>CLOSED</em></u> FOR MAINTENANCE!<br /><br />
-Be back soon!<br />
-IMPORTANT – PLEASE READ!</h1>
+# React SDK - Release Announcement
 
-<h3>On <strong>April 24, 2023</strong>, there will be a <strong>significant update</strong> to the React SDK <strong>main</strong> branch.</h3>
-<br />
+This is the latest version of the React SDK and it provides many new feature that are documented here:
```
Comment (JEAxero) [inline README.md:5]: make feature --> features

**Review by JEAxero — CHANGES_REQUESTED** [review]
Just a few minor changes

**Review by JEAxero — APPROVED** [review]
Looks good.

**Comment by niallriddell** [comment]
Done!!

**Pull request merged**